core-feature-pack/common/src/main/resources/content/bin/installation-manager.ps1

```diff
@@ -4,33 +4,23 @@
 param (
     [Parameter(Mandatory=$true)]
     [string]$installationHome,
-    [string]$instMgrLogProperties
+    [string]$instMgrLogProperties,
+    [string]$instMgrLogFile
 )
 
-# Sanitizes URLs to avoid issues with unsupported long paths
-function Sanitize-Path {
-    param(
-        [Parameter(Mandatory=$true)]
-        [string]$inputPath
-    )
-
-    # Check if the path starts with double backslashes and contains at least one additional character
-    if ($inputPath -match '^\\\\[^\\]+\\.*') {
-        $removedBackslash = $inputPath.Substring(2)
-        return "\\?\UNC\$removedBackslash"
-    } else {
-        return "\\?\$inputPath"
-    }
-}
+$TimeStamp = Get-Date -Format "yyyy-MM-dd HH:mm:ss"
```

Comment:
Same here as above, won't this always be set to the same timestamp?

Reply:
Fixed